### PR TITLE
DNN-19410

### PR DESCRIPTION
**Pull request by mean2me**

### CODE REVIEW

File: src/Modules/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/webpack.config.js [hunk 1]
```diff
@@ -60,7 +60,10 @@ module.exports = {
         })
     ] : [
             new webpack.DefinePlugin({
-                VERSION: JSON.stringify(packageJson.version)
+                VERSION: JSON.stringify(packageJson.version),
+                "process.env": {
+                    "NODE_ENV": JSON.stringify("development")
```
Comment (tpluscode) [inline src/Modules/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/webpack.config.js]: I don't think we really need this constant. It's enough to only ever check for `"production`"

**Review by tpluscode — CHANGES_REQUESTED** [review]
I did notice this some time ago but I cannot recall why we eventually kept the `reduxImmutableStateInvariant()` 🤔 

Also, I see it appears in pretty much [every single of the projects under this repository](https://github.com/search?q=org%3Adnnsoftware+reduxImmutableStateInvariant&type=Code). Should you change every `configureStore.js`?

**Comment by mean2me** [comment]
@tpluscode About *reduxImmutableStateInvariant* there many advices on the web about it. Using in a production environment overhead our application.

**Review by tpluscode — APPROVED** [review]

**Comment by tpluscode** [comment]
fixes #557

**Pull request merged**